### DISCUSSION TITLE
Just published! The boilerplate (starter) Tauri cross-platform desktop application with Aurelia 2 frontend and Rust backend

### DISCUSSION THREAD

**Discussion by glyad** [aurelia/aurelia#1876] [Show and tell]
Tauri is an alternative to Electron framework allowing to build an optimized, secure, and frontend-independent application for multi-platform deployment. Tauri built on Rust with performance and security at the center. 

Sources: [Aurelia 2 Tauri boilerplate app](https://github.com/glyad/au2-tauri-boilerplate/).

<img width="964" alt="image" src="https://github.com/aurelia/aurelia/assets/4238205/f9d2627c-8d97-4340-90ba-b4c245d38015">

**Comment by glyad** [comment]
@bigopon, I'll be happy to contribute it to Aurelia examples. It shows following use cases: Vite, easy of integration, and the IPC with Rust backend.

**Reply by bigopon** [reply]
Hi @glyad thanks for this offer, are you still interested doing this? I only realised where this thread is just now since we had a new discussion thread yesterday. Sorry for the delayed answer.

**Reply by glyad** [reply]
Hi @bigopon,
I'm also sorry, I missed your answer. Yes, I can continue with it. Just point me, how to...
